src/dsp_tools/xmllib/helpers.py

```diff
@@ -60,15 +61,12 @@ def create_label_to_name_list_node_mapping(
             }
         ]
         ```
-
-        ```python
-        result = xmllib.create_label_to_name_list_node_mapping(
-            project_json_path="project.json",
-            list_name="listName",
-            language_of_label="de",
-        )
-        # -> { "Knoten 1": "n1", "Knoten 2": "n2" }
-        ```
+        >>> xmllib.create_label_to_name_list_node_mapping(
+        ...     project_json_path="src/dsp_tools/xmllib/assets/project.json",
```

Comment:
This needs to reference a real file, otherwise it would fail. Alternatively, we could just mark this example with `# doctest: +SKIP`, then it wouldn't be tested.

Reply:
I don't like that you have to reference a real file. Now we have a file that we do not otherwise need and a path is just unnecessary noise. If we went with the docstest I would absolutely skip this.